rankers/LevenshteinRanker/__init__.py

```diff
@@ -17,13 +17,13 @@ class LevenshteinRanker(Match2DocRanker):
     required_keys = {"text"}
 
     def score(
-        self, query_meta: Dict, old_match_scores: Dict, match_meta: Dict
+        self, old_match_scores: Iterable[float], query_meta: Dict, match_meta: Iterable[Dict]
```

Comment:
Levenshtein changes have already been taken care for

Reply:
Yeah, I took your branch as it is so those changes were pushed too. I'm going to talk with Rutuja in half an hour on how to do the PRs for this